### PR TITLE
Add addresses.json files for both networks

### PR DESCRIPTION
**Pull request by weilbith**
Closes: #414

**Attention:**
As the referenced issue states, is the Trustlines Blockchain still missing the identity contracts. Therefore those values are empty. We must wait for them.

### CODE REVIEW

File: docker/trustlines/README.md [hunk 1]
```diff
@@ -31,10 +31,17 @@ We need to do some initial setup and configuration for the system to work. You
...
+the root of this repository. E.g. if you want to connect to the Trustlines
+Blockchain and index all registered currency networks of the foundation, you can
+copy it like that:
 
+```sh
+$ cp ../../config/addresses_tlbc.json ./addresses.json
+```
+
+This also includes the references to the identity related contracts. If you
+deploy your own contracts, please adapt `addresses.json` accordingly.
```
Comment (schmir-at-bb) [inline docker/trustlines/README.md]: I read this first like you could deploy your own identity contracts (which you can), but that's not necessary and will make the address recovery not work. Therefore please replace with:

If you deploy your own currency networks,... accordingly.

File: docker/trustlines/README.md [hunk 2]
```diff
@@ -31,10 +31,17 @@ We need to do some initial setup and configuration for the system to work. You
 need to provide the `addresses.json` file, which should be put in the directory
 alongside the `docker-compose.yml` file.
 
-This directory contains working example files for contracts already
-deployed on the laika blockchain. If you deploy your own contracts,
-please adapt `addresses.json` accordingly.
+You can find some prepared files for each network in the `config/` directory at
+the root of this repository. E.g. if you want to connect to the Trustlines
+Blockchain and index all registered currency networks of the foundation, you can
```
[the remark this answers is not in the document]
Comment (weilbith) [reply docker/trustlines/README.md]: d

File: docker/trustlines/README.md (hunk 1 above)
Comment (weilbith) [reply docker/trustlines/README.md]: :heavy_check_mark:

**Comment by weilbith** [comment]
The rebase solved the failing end2end tests. The new issue trustlines-network/project#816 is aware about the update for the missing identity contracts. Therefore this can be merged, as soon as the CI run through the cleaned history.

**Pull request merged**